src/main/java/com/jmb/springfactory/model/entity/ProductionOrder.java

```diff
@@ -0,0 +1,23 @@
+package com.jmb.springfactory.model.entity;
+
+import javax.persistence.Entity;
+
+import com.jmb.springfactory.model.enumeration.Measurements;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class ProductionOrder extends BaseEntity {
+
+  private String name;
+
+  private String description;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "description" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)